tests/ctl/core/test_api.py

```diff
@@ -590,6 +663,50 @@ async def test_system_create(
             cookie.name for cookie in systems[0].privacy_declarations[0].cookies
         ] == ["essential_cookie"]
         assert systems[0].privacy_declarations[1].cookies == []
+        system = systems[0]
+
+        assert system.vendor_id == "test_vendor"
```

Comment:
something similar here:

```python
for key, value in system.items():
    assert get_attr(system, key) == value
```

Reply:
compromise - dry up one set, leave system assertions as-is, but compare selected system response values to system?

Reply:
I'll take it! Thanks 🙂